code/modules/mob/living/carbon/human/human.dm

```diff
@@ -559,7 +560,17 @@
 		GLOB.badomens -= OMEN_NOLORD
 		priority_announce("The Ten have named [coronated.real_name] the inheritor of [SSmapping.config.map_name]!", \
 		title = "Long Live [lord_job.get_informed_title(coronated)] [coronated.real_name]!", sound = 'sound/misc/bell.ogg')
-
+	if(href_list[VV_HK_CHANGE_TITLE])
+		if(!src.mind?.assigned_role)
+			return
+		var/datum/job/human_job = src.mind.assigned_role
+		var/new_title = browser_input_text(usr, "What new title would you like to assign?", "Title Change")
+		if(!new_title)
+			return
+		human_job.title_override = new_title
```

Comment:
This changes the title for every job, its impossible to have individual overrides

Reply:
you would need mob level overrride like what previous advclass titles did but that kinda blows